lib/vsc/utils/zabbix.py

```diff
@@ -0,0 +1,64 @@
+
+# -*- encoding: utf-8 -*-
+#
+# Copyright 2012-2022 Ghent University
```

Comment:
```suggestion
# Copyright 2012-2022 Vrije Universiteit Brussel
```

Reply:
And run `find ./{lib,test} -type f -name '*.py' | REPO_BASE_DIR=$PWD xargs -I '{}' python -m vsc.install.headers '{}'` in the repo.

Reply:
this command reverts the change back to `Ghent University`